src/config/config.py

```diff
@@ -20,3 +20,4 @@ class Config:
     SQLALCHEMY_DATABASE_URI = 'postgresql://{}:{}@{}:{}/{}'.format(DB_USER, DB_PASSWORD,
                                                                    DB_HOSTNAME, DB_PORT, DB_NAME)
     USE_SESSION_FOR_NEXT = 1
+    UPLOAD_FOLDER = os.path.dirname(os.path.join(os.path.curdir, "../applicants"))
```

Comment:
`os.path.dirname(os.path.join(os.getcwd(), 'applicants'))`?

Reply:
A bit cleaner, if this is what you're shooting for

Reply:
aiming for src/applicants